### PR TITLE
use travis ci

### PR DESCRIPTION
**Pull request by rybit**

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -0,0 +1,12 @@
+---
+
+language: go
+
+go:
+  - 1.8
```
Comment (brycekahle) [inline .travis.yml]: I think this uses 1.8.0 instead of 1.8.3

File: .travis.yml [hunk 2]
```diff
@@ -0,0 +1,12 @@
...
+language: go
+
+go:
+  - 1.8.3
+
+install: make deps
+script: make test
+
+notifications:
+  email: false
```
Comment (brycekahle) [inline .travis.yml]: Why no emails? A user can turn them off if they don't want them.

Comment (rybit) [reply .travis.yml]: b/c I copied this from gotrue 😆

Comment (calavera) [reply .travis.yml]: they are very noisy

File: .travis.yml [hunk 3]
```diff
@@ -0,0 +1,12 @@
+---
+
+language: go
+
+go:
+  - 1.8.3
```
Comment (calavera) [inline .travis.yml]: let's add `1.9.x` to this list, so we build in both. 1.9 is the current stable.

**Pull request merged**